oceannavigator/frontend/src/components/LineWindow.jsx

```diff
@@ -312,29 +288,17 @@ class LineWindow extends React.Component {
       <Panel.Collapse>
         <Panel.Body>
           <DatasetSelector
-            key='dataset_0'
+            key='line_window_dataset_0'
```

Comment:
This `key` must be unique between all rendered `DatasetSelector`s so it's state gets updated correctly.